### ISSUE TITLE
Some information missing when compared to the standard JSDOC3 output.

### ISSUE THREAD

**Issue by cookch10** [julie-ng/tidy-jsdoc#11]
Given the following JavaScript:
```js
const TestClass = class TestClass {
  /**
   * Create a TestClass instance.
   * @class TestClass
   * @classdesc TestClass represents an object of some sort.
   * @author Bob Smith <bob@bob.com>
   * @param {(number|string)} value - Some value passed in.
   */
  constructor(value) {
    this._ctorValue = this.constructor.validationRegExp.test(value) ? value : 'someDefaultValue';
  }

  /**
   * Returns some arbitrary value given the supplied parameter.
   * @static
   * @param {string} value - A string.
   * @returns {string} Given the following input: "ABC" would return "A".
   */
  static getArbitraryValue(value) {
    return value[0];
  }


  /**
   * Returns a regular expression (RegExp) which indicates a valid TestClass value.
   * @static
   * @readonly
   * @property {regExp} validationRegExp
   */
  static get validationRegExp() {
    return /^\w+$/;
  }

  /**
   * Returns the arbitrary value from a valid TestClass value.
   * @instance
   * @readonly
   * @property {string} arbitraryValue - Given the following input: "ABC" would return "A".
   */
  get arbitraryValue() {
    return this.constructor.getArbitraryValue(this._ctorValue);
  }
};
```

When running this through the standard JSDOC3 template, compared to the tidy-jsdoc template, some fields appear to be missing.  Speficially, the `static` and `readonly` descriptors for members / properties.  In addition, some of the descriptive text is missing in the members / properties (also links to the source for the members / properties would be nice to have):

![standardjsdoc-compared-to-tidy-jsdoc-2020-02-18_1358](https://user-images.githubusercontent.com/6051695/74770329-073a0a80-525a-11ea-82e8-38ac807a80b4.png)

My JSDOC config is simple, and that looks like:
```js
  const jsDocConfig = {
    opts: {
      destination: './docs',
      template: '../node_modules/tidy-jsdoc'
    },
    metadata: {
      title: 'Some Example'
    }
  };
```

relevant package.json:
```json
{
  "jsdoc": "^3.6.3",
  "tidy-jsdoc": "^1.2.2"
}
```

Is this something which can be solved through a configuration, or would this require a change within tidy-jsdoc?

**Comment by julie-ng** [comment]
Thanks for reporting this. I have noted as a todo to check the following properties:

- `static`
- `readonly`

**Comment by julie-ng** [comment]
Fixed now in [v1.3.0 release](https://www.npmjs.com/package/tidy-jsdoc). If the problem persists, feel free to re-open this issue.
